packages/arb-token-bridge-ui/src/components/common/Dialog.tsx

```diff
@@ -98,10 +101,22 @@ export function Dialog(props: DialogProps) {
             props.children
           ) : (
             <>
-              <div className="px-8 py-4">
+              <div className="relative px-8 py-4">
                 <HeadlessUIDialog.Title className="text-2xl font-medium">
                   {props.title}
                 </HeadlessUIDialog.Title>
+                {props.closeable && (
+                  <button
+                    type="button"
+                    className="absolute top-5 right-10"
+                    onClick={() => props.onClose(false)}
+                  >
+                    <img
+                      src={CloseIcon}
+                      className="h-5 w-5 hover:opacity-70"
```

Comment:
We have an `arb-hover` class commonly used for the hover effects. Also, the `img` is missing an alt attribute.